README.md

```diff
@@ -1 +1,32 @@
-# riscv
+﻿RV32G Processor model
+===================================================================
+
+This is the functional model of RISC-V instruction set architecture. It contains all the instructions necessary for running mibench programs. It has following types of instruction encoding:
```

Comment:
I tried making the line break after 80 characters. In my editor it breaks but as soon as I upload ot on git that format goes away.